charts/openfga/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: postgresql
-  repository: https://charts.bitnami.com/bitnami
+  repository: oci://registry-1.docker.io/bitnamicharts
```

Comment:
switching to OCI might have some unintended consequences, for instance this is generaly available only starting at helm v3.8.0 and some of our CI uses 3.5.0 (https://github.com/openfga/helm-charts/blob/830d1abe38d4e7a6241f5f614982f92db6a38b82/.github/workflows/release.yml#L36) 

I do agree that this change is a nice to have but maybe best to include it in a different PR to make sure this one contains only changes related to that datastores configuration, and that if we have to revert it for any reason we don't remove the part you're most interested in :)

Reply:
This was a change I needed because of downtime on bitnami charts at time of a critical deployment. I'll remove these things next week and clean everything up